announcements/move-to-cluster-api.md

```diff
@@ -0,0 +1,42 @@
+- Significantly more flexibility and configurability over your infrastructure. Things like deploying into existing networks or ... will now be possible.
+- A future-proof API that protects you from vendor lock-ins, developed by a dynamically growing community which will increase the speed of adding new functionality over time.
+- The possibility to use all tooling compatible with the Cluster API -- not only that created by Giant Swarm.
+- Ability to create and manage workload clusters with managed control planes like EKS or AKS.
+- Adding new providers will become easier and Giant Swarm will be in a better position to expand their offering beyond AWS, Azure, and current On-Premises solutions.
+- The possibility to have workload clusters from different providers managed by one management cluster to enable multi-cloud use cases.
+
+### What will the move look like?
+
+Giant Swarm is investing an enormous amount of effort to make the process safe and convenient for you. From your perspective, it will look like a regular update to a new major version -- no migration will be required. On the Giant Swarm side, a series of well-tested, automated actions will be performed to transform your current clusters into the Cluster API ones. Giant Swarm wants to provide a transition path for the clusters in all versions and transform them into Cluster API clusters with little to no downtime in the Kubernetes control plane and no downtime in your workloads. Giant Swarm will perform an in-place upgrade to preserve your current IP addresses.
+
+
+In the first phase, starting the transition will be enabled for Giant Swarm personnel only, but eventually, you will be able to perform it on your workload clusters autonomously.
+
+### What are the risks?
+
+The infrastructure behind Giant Swarm clusters differs in many ways from the one backing a Cluster API cluster. Giant Swarm engineers worked hard to find an automated, reliable, and reproducible transition path between the two, but there is always the risk that something goes wrong. Giant Swarm will ensure a well-defined, thoroughly tested transition plan to mitigate the risks and have a recovery plan in place if something unexpected happens. The transition will also be done in phases, leaving your mission-critical cluster to the end to adapt the process to all possible custom configurations and make it bullet-proof.
+
+
+### What will change?
+
+There will be very little to no differences between the Giant Swarm and the Cluster API clusters from the functional point of view. After the transition, you will be able to use the upstream Cluster API tools to manage the clusters.
+
+The most noticeable change will be the deprecation of the 'gsctl' tool that will no longer work against Cluster API clusters. You will be able to continue using Giant Swarm Web UI 'Happa' and 'kubectl gs' plugin but only through your company's Single Sign On. 
```

Comment:
Maybe clarify that this is meant to be towards the API of the MC and has nothing to do with how you talk to the WC API

Reply:
I'm sorry I don't know how to phrase that.
Does it mean that gsctl can still be used for WCs? or it wasn't used for that at all?

Reply:
the thing is not gsctl, it's that customer know they can use SSO on WCs and might confuse things here, I'll add sth to your suggestion below